web/api/py/codechecker_api/setup.py

```diff
@@ -8,7 +8,7 @@
 with open('README.md', encoding='utf-8', errors="ignore") as f:
     long_description = f.read()
 
-api_version = '6.39.0'
+api_version = '6.41.0'
```

Comment:
Side comment: previously you incremented the patch version number because you didn't see the changes after install. I created a patch and I put a warning note to the end of the readme file what to do in this case. For more info see: #3319